module/Finna/src/Finna/Controller/R2SearchController.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * R2 Search Controller
+ *
+ * PHP version 7
+ *
+ * Copyright (C) The National Library of Finland 2020.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Controller
+ * @author   Samuli Sillanpää <samuli.sillanpaa@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:plugins:controllers Wiki
+ */
+namespace Finna\Controller;
+
+/**
+ * R2 Search Controller
+ *
+ * @category VuFind
+ * @package  Controller
+ * @author   Samuli Sillanpää <samuli.sillanpaa@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org/wiki/development:plugins:controllers Wiki
+ */
+class R2SearchController extends SearchController
+{
+    use \Finna\Controller\R2ControllerTrait;
+
+    /**
+     * Search class family to use.
+     *
+     * @var string
+     */
+    protected $searchClassId = 'R2';
+
+    /**
+     * Results action.
+     *
+     * @return mixed
+     */
+    public function searchAction()
+    {
+        $view = parent::resultsAction();
```

Comment:
Mikä tämän tarkoitus on? Jos pitää olla, niin jokin kommentti, miksi on olemassa.

Reply:
Poistin tämän ja muutin routen > R2/Results. Näin on yhtenäinen L1:n kanssa.